### PR TITLE
Python37

### PR DESCRIPTION
**Pull request by cgoldberg**
This PR adds Python 3.7 support

### CODE REVIEW

**Comment by cgoldberg** [comment]
Travis CI slave doesn't have 3.7 installed

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ language: python
 cache: pip
 matrix:
   include:
+    - python: 3.7
```
Comment (mbeacom) [inline .travis.yml]: @cgoldberg You can get travis to run this via something like:

```yaml
  include:
    - python: 3.7
      env: TOXENV=py37
      dist: xenial
      sudo: true
```

**Comment by mbeacom** [comment]
@cgoldberg It should pass if you add `sudo: true` to the `py37` travis matrix - https://travis-ci.org/mbeacom/locust/jobs/430394601

**Comment by cgoldberg** [comment]
@mbeacom thanks!  merging this now

**Pull request merged**